files/default/sql/ddl/2.6.0__initial_tables.sql

```diff
@@ -2270,3 +2270,65 @@ CREATE TABLE IF NOT EXISTS `git_repository_remotes` (
                                           KEY `git_repository_fk` (`repository`),
                                           CONSTRAINT `git_repository_fk` FOREIGN KEY (`repository`) REFERENCES `git_repositories` (`id`) ON DELETE CASCADE
 ) ENGINE=ndbcluster AUTO_INCREMENT=6164 DEFAULT CHARSET=latin1 COLLATE=latin1_general_cs;
+
+CREATE TABLE IF NOT EXISTS `expectation_suite` (
+    `id` INT(11) NOT NULL AUTO_INCREMENT,
+    `feature_group_id` INT(11) NOT NULL,
+    `name` VARCHAR(63) NOT NULL,
+    `meta` VARCHAR(1000) DEFAULT "{}",
+    `data_asset_type` VARCHAR(50),
+    `ge_cloud_id` VARCHAR(200),
+    `run_validation` BOOLEAN DEFAULT TRUE,
+    `validation_ingestion_policy` VARCHAR(25),
+    `created_at` timestamp DEFAULT CURRENT_TIMESTAMP,
+    PRIMARY KEY (`id`),
+    CONSTRAINT `feature_group_suite_fk` FOREIGN KEY (`feature_group_id`) REFERENCES `feature_group` (`id`) ON DELETE CASCADE ON UPDATE NO ACTION
+) ENGINE=ndbcluster DEFAULT CHARSET=latin1 COLLATE=latin1_general_cs;
+
+CREATE TABLE IF NOT EXISTS `expectation` (
```

Comment:
This pattern of having a row per expectation, inside an expectation suite, where the 
expectations are not reused over expectation_suites, will mean it is slow to retrieve
100s of expectations. I will not be surprised if we have to refactor to add all expectations as a 
JSON column inside the expectation suite. 
However, for now, we can let it go.

Reply:
This is faster than having one big json saved on HopsFS and having to deserialize it.